bescort.lic

```diff
@@ -301,6 +302,39 @@ class Bescort
 
   private
 
+  def use_flying_mount(type, mode, speed = 'fly')
+    case mode
+    when /^mount/
+      bput("get my #{@flying_mount}", /You get /, /You are already holding/)
+      case type
+      when /broom/i
```

Comment:
it might be worth just downcasing the `type` at the start of the method.  Then you can do string matching here `/broom/i` -> `'broom'`

Reply:
Done for the mode, which makes the change from:
`/^mount/` -> `'mount'`  and `/^dismount/i` -> `'dismount'`

But I'm leaving the mount type variable as is because I want to encourage people to enter their mount with both adjective and noun and I only am trying to match whether it's a carpet or broom.

ie: `flying_mount: silk carpet`
so `/carpet/i` will match the yaml setting no matter the adjective and also even if they use any capital letters.